src/plugins/intel_cpu/tests/functional/behavior/export_import.cpp

```diff
@@ -35,34 +37,91 @@ std::shared_ptr<ov::Model> MakeMatMulModel() {
+
+    auto new_network = core.compile_model(original_model, device_name, new_properties_input);
+    auto new_properties_output = GetProperties(new_network);
+
+    std::stringstream exported_model;
+    original_network.export_model(exported_model);
+
+    // import_model with original config can create the same multi_thread setting as compile_model
     {
-        std::stringstream ss(exported_stream.str());
-        auto imported_tp_network = core.import_model(ss, deviceName, tput_mode);
-        auto nstreams_tp_imported = imported_tp_network.get_property(ov::num_streams.name()).as<std::string>();
-        EXPECT_EQ(nstreams_tp_original, nstreams_tp_imported);
+        std::stringstream ss(exported_model.str());
+        auto imported_network = core.import_model(ss, device_name, original_properties_input);
+        auto imported_properties_output = GetProperties(imported_network);
+
+        EXPECT_EQ(original_properties_output[0], imported_properties_output[0]);
+        EXPECT_EQ(original_properties_output[1], imported_properties_output[1]);
+        EXPECT_EQ(original_properties_output[2], imported_properties_output[2]);
+        EXPECT_EQ(original_properties_output[3], imported_properties_output[3]);
+        EXPECT_EQ(original_properties_output[4], imported_properties_output[4]);
+        EXPECT_EQ(original_properties_output[5], imported_properties_output[5]);
+        EXPECT_EQ(original_properties_output[6], imported_properties_output[6]);
     }
 
+    // import_model with new properties can create the same multi_thread setting as compile_model with new properties
     {
-        std::stringstream ss(exported_stream.str());
-        auto imported_latency_network = core.import_model(ss, deviceName, latency_mode);
-        auto nstreams_latency_imported = imported_latency_network.get_property(ov::num_streams.name()).as<std::string>();
-        EXPECT_EQ(nstreams_latency_original, nstreams_latency_imported);
+        std::stringstream ss(exported_model.str());
+        auto imported_network = core.import_model(ss, device_name, new_properties_input);
+        auto imported_properties_output = GetProperties(imported_network);
+
+        EXPECT_EQ(new_properties_output[0], imported_properties_output[0]);
+        EXPECT_EQ(new_properties_output[1], imported_properties_output[1]);
+        EXPECT_EQ(new_properties_output[2], imported_properties_output[2]);
+        EXPECT_EQ(new_properties_output[3], imported_properties_output[3]);
+        EXPECT_EQ(new_properties_output[4], imported_properties_output[4]);
+        EXPECT_EQ(new_properties_output[5], imported_properties_output[5]);
+        EXPECT_EQ(new_properties_output[6], imported_properties_output[6]);
     }
 }
 
-INSTANTIATE_TEST_CASE_P(smoke_ExportImportTest, ExportOptimalNumStreams, ::testing::Values(std::string("CPU")));
+const std::vector<std::vector<ov::AnyMap>> testing_properties = {
+    {{ov::num_streams(1)}, {ov::num_streams(2)}},
```

Comment:
Maybe I missed smt, but I don't see coverage of the followig case:  num_streams/num_threads is not provided inside the compilation config -> plugin automatically computes optimal number of streams based on the model -> import model also not specifying num streams via config -> num streams is equal with original model.

Reply:
I have modified test cases, only test one property at a time.